package.json

```diff
@@ -1,12 +1,16 @@
 {
-  "name": "@pmedianetwork/eslint-config",
-  "version": "3.2.0",
+  "name": "eslint-plugin-pmedianetwork",
+  "version": "4.0.0",
```

Comment:
no need to change the version manually - it will be done with the automated build process. If we have version 4 here now, version 5 will be released!

Reply:
https://github.com/pmedianetwork/eslint-config/pull/64/commits/9c05bc03230a0283e513fdd59ec9868051d9038e

Reply:
Ok - so they bump automatically - awesome!